pkgs/development/web/cypress/default.nix

```
@@ -39,7 +40,7 @@ in stdenv.mkDerivation rec {
   # don't remove runtime deps
   dontPatchELF = true;
 
-  nativeBuildInputs = [ autoPatchelfHook wrapGAppsHook3 unzip ];
+  nativeBuildInputs = [ autoPatchelfHook wrapGAppsHook3 unzip makeWrapper];
```

```suggestion
  nativeBuildInputs = [ autoPatchelfHook (wrapGAppsHook3.override { makeWrapper = makeShellWrapper; }) unzip makeShellWrapper];
```

Specify `makeShellWrapper` because only it has `--run` `makeBinaryWrapper` doesn't have `--run`

`wrapGAppsHook3` brings in `makeBinaryWrapper`

---

Info about why there's no `--run` for `makeBinaryWrapper` https://github.com/NixOS/nixpkgs/pull/124556#discussion_r760628535

---

```
> Wrapping link: '/nix/store/ham2acsq0fmyl72vpsza6xhs8li9pz9q-cypress-13.13.2/bin/cypress'
> checking for references to /build/ in /nix/store/ham2acsq0fmyl72vpsza6xhs8li9pz9q-cypress-13.13.2...
> patchelf: cannot find section '.dynamic'. The input file is most likely statically linked
> patching script interpreter paths in /nix/store/ham2acsq0fmyl72vpsza6xhs8li9pz9q-cypress-13.13.2
> /nix/store/ham2acsq0fmyl72vpsza6xhs8li9pz9q-cypress-13.13.2/opt/cypress/resources/app/node_modules/jake/bin/bash_completion.sh: interpreter direc…
> /nix/store/ham2acsq0fmyl72vpsza6xhs8li9pz9q-cypress-13.13.2/opt/cypress/resources/app/node_modules/open/xdg-open: interpreter directive changed f…
> /nix/store/ham2acsq0fmyl72vpsza6xhs8li9pz9q-cypress-13.13.2/opt/cypress/resources/app/node_modules/fluent-ffmpeg/tools/test-travis.sh: interprete…
> /nix/store/ham2acsq0fmyl72vpsza6xhs8li9pz9q-cypress-13.13.2/opt/cypress/resources/app/node_modules/better-sqlite3/deps/download.sh: interpreter d…
> stripping (with command strip and flags -S -p) in  /nix/store/ham2acsq0fmyl72vpsza6xhs8li9pz9q-cypress-13.13.2/bin
> /nix/store/91d27rjqlhkzx7mhzxrir1jcr40nyc7p-stdenv-linux/setup: eval: line 228: unexpected EOF while looking for matching `''
```